homeassistant/components/alarm_control_panel/strings.json

```
@@ -62,5 +62,77 @@
         }
       }
     }
+  },
+  "services": {
+    "alarm_disarm": {
+      "name": "Disarm",
+      "description": "Disarms the alarm.",
+      "fields": {
+        "code": {
+          "name": "Code",
+          "description": "Code to disarm the alarm."
+        }
+      }
+    },
+    "alarm_arm_custom_bypass": {
+      "name": "Arm with custom bypass",
+      "description": "Send arm custom bypass command.",
+      "fields": {
+        "code": {
+          "name": "[%key:component::alarm_control_panel::services::alarm_disarm::fields::code::name%]",
+          "description": "Code to arm the alarm."
+        }
+      }
+    },
+    "alarm_arm_home": {
+      "name": "Arm home",
+      "description": "Send the alarm the command for arm home.",
+      "fields": {
+        "code": {
+          "name": "[%key:component::alarm_control_panel::services::alarm_disarm::fields::code::name%]",
+          "description": "[%key:component::alarm_control_panel::services::alarm_arm_custom_bypass::fields::code::description%]"
+        }
+      }
+    },
+    "alarm_arm_away": {
+      "name": "Arm away",
+      "description": "Send the alarm the command for arm away.",
```

```suggestion
      "description": "Sets  the alarm to: a_rmed, no one home_.",
```
probably a clumsy way of saying this. But at first I did not understand the difference between arm home and arm away.